tensorflow/python/kernel_tests/map_ops_test.py

```diff
@@ -180,5 +180,25 @@ def testSameKeyInsertLookupGrad(self):
       self.assertAllClose(g, array_ops.zeros_like(v))
       self.assertAllClose(g2, 5)
 
+  def testSameKeyInsertLookupGrad2(self):
```

Comment:
I think we can combine the tests, or otherwise can we give the two a suggestive name?

Reply:
Done!